operator/pkg/controller/workspace/tfc_output.go

```diff
@@ -98,23 +108,29 @@ func convertValueToString(val cty.Value) string {
 		sort.Strings(attrNames)
 
 		var b bytes.Buffer
-		b.WriteString("{")
-
 		i := 0
 		for _, attr := range attrNames {
+			k := attr
```

Comment:
Could you please elaborate on why this indirection of `attr` through `k` was necessary? I'm guessing it's for aligning to the `k, v` naming convention, but I might be missing the point.

Reply:
Hey @alexsomesan , yes, just naming convention.  I could totally keep it as attr.  What do you think?

Reply:
I guess either one of them would be ok, but I don't see a good reason for having both. You could replace `attr` with `k` in the for statement but I feel `k` is less descriptive than `attr` when trying to comprehend the code.